src/components/scenes/GuiPluginListScene.js

```diff
@@ -107,6 +112,18 @@ class GuiPluginList extends React.PureComponent<Props, State> {
     }
   }
 
+  async updatePlugins() {
+    try {
+      const response = await fetchInfo(`v1/buySellPlugins/${config.appId ?? 'edge'}`)
+      const reply = await response.json()
+      const plugins = asBuySellPlugins(reply)
+      this.setState({ buySellPlugins: plugins })
+    } catch (e) {
+      console.log(e.message)
+      // This is ok. We just use default values
+    }
+  }
+
```

Comment:
This works, but we don't have it cached, so we either need to add a TODO or we need to add a cache. Otherwise, the user will see the old list each time they visit the scene, only to have it flash to the new list after the network gets back.

Reply:
That's fair. What's the correct `account.dataStore` equivalent for local (non-synced) storage that I can use?

Reply:
We have a task to create this. Since it is not synced, the state.core.disklet should be fine. This is app-global, not tied to any account.